src/System.Net.Security/tests/FunctionalTests/ClientAsyncAuthenticateTest.cs

```diff
@@ -27,20 +27,14 @@ public class ClientAsyncAuthenticateTest
 
         private static readonly SslProtocols[] s_eachSslProtocol = new SslProtocols[]
         {
-            SslProtocols.Ssl2,
```

Comment:
I think ClientAsyncAuthenticate_EachProtocol_Ssl2_Success is the solution to my question above. If that's true, please ignore the comment above.